### PR TITLE
chore(Datagrid): datagrid column customization fix

### PR DESCRIPTION
**Pull request by Ratheeshrajan**
Contributes to #4244 
1. Copyright header in every source file (js, css, scss etc.) with the appropriate years.
2. All components utilizing motion must include reduced-motion queries for
accessibility purposes.

#### What did you change?

1. Updated Copyright header in 10 files to include the current year.
2. No transition in reduced motion

#### How did you test and verify your work?
Files change

### CODE REVIEW

File: packages/ibm-products-styles/src/components/Datagrid/styles/_draggableElement.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 //
-// Copyright IBM Corp. 2022, 2022
+// Copyright IBM Corp. 2022, 2022, 2023, 2024
```
Comment (matthewgallo) [inline packages/ibm-products-styles/src/components/Datagrid/styles/_draggableElement.scss:2]: The format/pattern for the years in the copyright headers is year file was created followed by year the file was last edited, so you only will have two years listed.

**Review by matthewgallo — CHANGES_REQUESTED** [review]
See comment regarding the pattern we use for the years listed in the copyright header (tldr `year created, year last updated`) so you'll only end up with two years listed.

File: packages/ibm-products-styles/src/components/Datagrid/styles/_draggableElement.scss (hunk 1 above)
Comment (Ratheeshrajan) [reply packages/ibm-products-styles/src/components/Datagrid/styles/_draggableElement.scss:2]: Thanks @matthewgallo for the information. So I can update the year last edited as `2023`, right?
Screenshot 2024-02-23 at 10 58 16 AM

Comment (matthewgallo) [reply packages/ibm-products-styles/src/components/Datagrid/styles/_draggableElement.scss:2]: Correct ^^

Comment (Ratheeshrajan) [reply packages/ibm-products-styles/src/components/Datagrid/styles/_draggableElement.scss]: @matthewgallo updated the copyright header `last modified year`.

**Review by elycheea — APPROVED** [review]

**Review by matthewgallo — APPROVED** [review]

**Pull request merged**